core/src/syscall/write.rs

```diff
@@ -56,7 +56,7 @@ impl Syscall for SyscallWrite {
                         u32_to_comma_separated(rt.state.global_clk - start)
                     );
                 } else {
-                    log::info!("stdout: {}", s.trim_end());
+                    log::debug!("stdout: {}", s.trim_end());
```

Comment:
I think this should be changed back to `log::info!()`?

Reply:
fixed